docs/topics/rn-new-features.adoc

```diff
@@ -7,25 +7,25 @@
 
 This section describes the new features of the {ProductName} ({ProductShortName}) {ProductVersion}.
 
+[IMPORTANT]
+====
+Rules for migrating from Java EE8 to Jakarta EE9 and OpenRewrite support are both provided as Technology Preview only. Technology Preview features are not supported with Red Hat production service level agreements (SLAs), might not be functionally complete, and Red Hat does not recommend to use them for production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
+
+See link:{KBArticleTechnologyPreview}[Technology Preview features support scope] on the Red&nbsp;Hat Customer Portal for information about the support scope for Technology Preview features.
+====
+
 .Rules for migrating from Java EE8 to Jakarta EE9
 {ProductShortName} rules replace Java EE8 artifact dependencies and package import statements and rename XML schema namespaces, prefixed properties, and bootstrapping files for Jakarta EE9.
 
-Most rules contain quick fixes to automate source code changes when using an {ProductShortName} IDE plugin or extension for Eclipse, Eclipse Che, IntelliJ IDEA, Microsoft Visual Studio Code, or Red Hat CodeReady Studio.
+You run these rules by specifying the `jakarta-ee` target.
 
-To execute the rules, use the target `jakarta-ee`.
+Most rules contain quick fixes to automate source code changes when using an {ProductShortName} IDE plugin or extension for Eclipse, Eclipse Che, IntelliJ IDEA, Microsoft Visual Studio Code, or Red Hat CodeReady Studio.
```

Comment:
It's not clear to me whether this sentence applies to the new rules or not. It looks like a general description of rules and probably does not belong in the RN for this new feature, since the new feature is the Java > Jakarta rules.